src/main/java/uk/ac/cam/cl/dtg/segue/api/QuestionFacade.java

```diff
@@ -437,4 +529,13 @@ public Response generateSpecification(@Context final HttpServletRequest request,
       return responseWrapper.toResponse();
     }
   }
+
+  // Move these out of the facade
+  private static List<String> splitCsvStringQueryParam(final String queryParamCsv) {
+    if (null != queryParamCsv && !queryParamCsv.isEmpty()) {
+      return Arrays.asList(queryParamCsv.split(","));
+    } else {
+      return null;
```

Comment:
Have you considered returning an empty array? It would mean that the result of this function will never be null, which makes it easier to use, as you know fore sure that the result of this function is always a list.